eng/tools/BaselineGenerator/BaselineGenerator.csproj

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
-    <StartArguments>-s https://api.nuget.org/v3/index.json</StartArguments>
```

Comment:
This way, we can pass in a source via command line. This is redundant anyway since the nuget feed is used by default: https://github.com/aspnet/AspNetCore/blob/master/eng/tools/BaselineGenerator/Program.cs#L50